ui/actions/nuxeo-preview-button.js

```diff
@@ -155,7 +155,8 @@ import '../nuxeo-button-styles.js';
     }
 
     _isAvailable(document) {
-      return !(this.hasFacet(document, 'Folderish') || this.hasFacet(document, 'Collection'));
+      return !(this.hasFacet(document, 'Folderish') || this.hasFacet(document, 'Collection')) ||
```

Comment:
Wouldn't it be better to have the `_isAvailable` method use `hasContent(this.document, this.xpath)` instead? Checking that a document has attachments only makes sense if the xpath is `files:files`, right?

Reply:
Actually, after our discussion yesterday, I thought about having both conditions `hasContent(document, this.xpath) && hasAttachments(document)`, but you are right. It makes more sense to rely on filtering for the blob.